specification/schemas/ZoneResponse.json

```
@@ -0,0 +1,40 @@
+{
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/Zone"
+    },
+    {
+      "required": [
+        "id",
+        "attributes"
+      ],
+      "properties": {
+        "attributes": {
+          "required": [
+            "postal_code",
+            "country_code",
+            "state_code",
+            "code",
+            "name",
+            "description"
```

I don't think all zones will have a `state_code` or `postal_code`, but they will have a `created_at`.
```suggestion
            "country_code",
            "code",
            "name",
            "created_at"
```